src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/cropping-slide.js

```diff
@@ -0,0 +1,516 @@
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+/**
+ * @class CroppingSlide
+ * Class which handles the cropping-slide in the media-edit overlay.
+ * Note that this is not an aura-component but a simple require-js dependency.
```

Comment:
There is no need for the cropping-slide to have it's own sandbox, as it does not provide custom events for example. Moreover the cropping-slide is only valid in the context of the overlay. Making it as a require module allowed me to also outsource the slide-definition from the overlay to the cropping-slide.